corehq/apps/app_manager/helpers/validators.py

```diff
@@ -1154,6 +1154,8 @@ def check_actions(self):
             subcase_names=subcase_names
         ))
 
+        errors.extend(self.check_for_conflicting_questions())
```

Comment:
What do you feel about combining this line and the above line into one extend?

Reply:
Can you explain the motivation for this change? To me, the existing code already handles exceptions in 3 distinct ways:
1) lack of subcase case types
2) lack of case name
3) invalid property names

This PR is adding a 4th condition:
4) duplicate case properties

Conceptually, that seems on the same level as the first 3, and it doesn't seem like it would naturally fit into the "invalid property names" check. Because of that, I think its more readable to leave this as its own statement